janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -90,7 +90,7 @@
         mapWriter = mapper.writerWithView(Map.class);
     }
 
-    private static final ElasticMajorVersion DEFAULT_VERSION = ElasticMajorVersion.FIVE;
+    private static final ElasticMajorVersion DEFAULT_VERSION = ElasticMajorVersion.SIX;
```

Comment:
I would directly upgrade to default version 7 or?

Reply:
`DEFAULT_VERSION` is used only when we cannot determine the version from ES which can potentially happen only if we are using unsupported versions or custom ElasticSearch builds. I am OK with any default version, so I've updated it to `SEVEN` as you propose.

Reply:
Thank you for the update.